### PR TITLE
Grant: Add storybook template

### PR DESCRIPTION
**Pull request by waitingallday**
**Issue**:

### What I did

- Add mocked SFC for grant content type
- Had to switch to declaring components imperatively (can't auto import outside nuxt)
- Fix a few styles and tests
- 

### How to test

- 
- 

#### For all PR's

- [ ] I've added relevant changes to the project Readme if needed.
- [ ] I've updated the documentation site as needed.
- [ ] I have added unit tests to cover my changes (if not applicable, please state why in a comment)

#### For new components only

- [ ] I have added a story covering all variants
- [ ] I have checked a11y tab in storybook passes
- [ ] Any events are emitted on the event bus

### CODE REVIEW

File: packages/ripple-ui-core/src/components/expandable/expandable.vue [hunk 1]
```diff
@@ -50,7 +50,7 @@ watch(
   </div>
 </template>
 
-<style scoped>
```
Comment (waitingallday) [inline packages/ripple-ui-core/src/components/expandable/expandable.vue:53]: This was causing trouble on the mocked grant template in storybook - the styles weren't rendered. It worked fine under nuxt though

File: packages/ripple-tide-grant/src/grant.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import {
+  Canvas,
+  Meta,
+  Story
+} from '@storybook/addon-docs'
+
+import fixture from '../../../examples/nuxt-app/test/fixtures/grant/tc-9a-grant-simple-test-date-range.json'
```
Comment (waitingallday) [inline packages/ripple-tide-grant/src/grant.stories.mdx:7]: Using the same fixture as the cypress tests 🙌

File: packages/ripple-tide-grant/src/grant.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+import MockPage from '../../ripple-ui-core/stories/components/mock-page.vue'
```
Comment (waitingallday) [inline packages/ripple-tide-grant/src/grant.stories.mdx:8]: Abstracted this enough to reuse with other content types

**Comment by dylankelly** [comment]
Merge after #253

**Pull request merged**